packages/RichTextEditing-Tests.package/RichTextEditorTests.class/instance/testInspectDocument.st

```diff
@@ -0,0 +1,10 @@
+unit-tests
+testInspectDocument
+	
+	| possibleMorphs |
+	self editorModel inspectDocument.
+	possibleMorphs := Project current world submorphs.
+	possibleMorphs := possibleMorphs select: [:morph | morph class = PluggableSystemWindow].
+	possibleMorphs := possibleMorphs select: [:morph | morph label = self document class name].
+	possibleMorphs := possibleMorphs select: [:morph | morph model object = self editorModel targetDocument].
+	self assert: (possibleMorphs notEmpty)
```

Comment:
Explain pls

Reply:
Check if the Document Inspector is opened. 
Inspect all open Morphs. Select for pluggableSystemWindow, of which Inspector is an object. Other morphs may not understand the message label. Then select the window which is the correct inspect Document